CHANGELOG.md

```diff
@@ -2,6 +2,7 @@ x.y.z Release notes (yyyy-MM-dd)
 =============================================================
 ### Enhancements
 * Throw an exception for Objects that have none of its properties marked with @objc.
+* Mac Catalyst is now supported when integrating with Cocoapods.
```

Comment:
and arm64 simulators

Also need to mention that the minimum version of cocoapods supported is now 1.10